pkg/grpc/device_definition.proto

```diff
@@ -143,6 +143,7 @@ message Integration {
   int32 auto_pi_default_template_id = 5;
   AutoPiPowertrainTemplate auto_pi_powertrain_template = 6;
   int32 refresh_limit_secs = 7;
+  uint64 token_id = 8;  // token_id can have a 0 value, which means it has not yet been minted and no token id has been assigned. This case should be checked for and handled. 
```

Comment:
Does this show up in the generated source? My side goal here is that if we look at the field in an IDE it will show this hint.

Reply:
Yes it does

Reply:
That's good. Can I be picky and ask for it above the variable? That seems more standard, e.g.,

https://github.com/googleapis/googleapis/blob/master/google/cloud/bigquery/storage/v1beta2/protobuf.proto